hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseStoreProvider.java

```diff
@@ -52,7 +52,8 @@ public String version() {
          * [1.0] HugeGraph-1328: supports backend table version checking
          * [1.1] HugeGraph-1322: add support for full-text search
          * [1.2] #296: support range sortKey feature
+         * [1.3] #287: Support pagination when do index query
```

Comment:
update to:
"Support pagination for index query" or "Support pagination when doing index query"